includes/api/shipping-info.php

```diff
@@ -638,6 +638,17 @@ function smartCodRestriction($addresses, $order)
     return true;
 }
 
+function codTrim($str){
+
+  $str=strtoupper($str);
```

Comment:
The function name tells as it's job is trimming, but why are we setting it to upper case? @poojarazor

Reply:
replaced by trim